Makefile

```diff
@@ -624,14 +624,18 @@ run-build-vm: $(BIOS_IMG) $(DEVICETREE_DTB)
 	$(QEMU_SYSTEM) $(QEMU_OPTS) -drive format=qcow2,file=$(BUILD_VM)
 
 run-live-vb:
-	@[ -f "$(LIVE).vdi" ] || { echo "Please run: make live-vdi"; exit 1; }
+	@[ -f "$(CURRENT_DIR)/live.vdi" ] || { echo "Please run: make live-vdi"; exit 1; }
 	VBoxManage list vms | grep $(VB_VM_NAME) >/dev/null &&  VBoxManage controlvm $(VB_VM_NAME) acpipowerbutton & sleep 10 & VBoxManage unregistervm $(VB_VM_NAME) --delete || echo "No VMs with $(VB_VM_NAME) name"
 	VBoxManage createvm --name $(VB_VM_NAME) --register --basefolder $(DIST)/
 	VBoxManage modifyvm $(VB_VM_NAME) --cpus $(VB_CPUS) --memory $(VB_MEMORY) --vram 16 --nested-hw-virt on --ostype Ubuntu_64  --mouse usbtablet --graphicscontroller vmsvga --boot1 disk --boot2 net
 	VBoxManage storagectl $(VB_VM_NAME) --name "SATA Controller" --add SATA --controller IntelAhci --bootable on --hostiocache on
-	VBoxManage storageattach $(VB_VM_NAME)  --storagectl "SATA Controller" --port 0 --device 0 --type hdd --medium $(LIVE).vdi
-	VBoxManage modifyvm $(VB_VM_NAME) --nic1 natnetwork --nat-network1 natnet1 --cableconnected1 on --natpf1 ssh,tcp,,$(SSH_PORT),,22
+	VBoxManage storageattach $(VB_VM_NAME)  --storagectl "SATA Controller" --port 0 --device 0 --type hdd --medium $(CURRENT_DIR)/live.vdi
+	# Create NAT networks if they don't exist
+	VBoxManage natnetwork add --netname natnet1 --network "$(QEMU_OPTS_NET1)" --enable 2>/dev/null || true
+	VBoxManage natnetwork add --netname natnet2 --network "$(QEMU_OPTS_NET1)" --enable 2>/dev/null || true
```

Comment:
`$(QEMU_OPTS_NET2)`
I see it's fixed in the second commit, but it looks confusing when checking per commit.